### PR TITLE
English book admonish headers take 1

### PR DESCRIPTION
**Pull request by PEZ**
I've added the title **More information** to all `info` admonishes, and **Please note!** for all `note`s. These are our fallbacks.

Then I have updated the `info` and `note` admonishes with tailored titles for the README and ch1 - ch4 (inclusive). Not sure you agree with my choice of tone, so thinking you can have a say before I barge ahead with the rest of the book. 😄

### CODE REVIEW

File: macroquad-introduction-book-english/src/README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 This guide is written by Olle Wreede at [Agical](https://www.agical.se/).
 
 <div class="onlyprint">
-This guide is available online at the following address:
```
Comment (ollej) [inline macroquad-introduction-book-english/src/README.md:8]: I guess this  was removed automatically, but the two spaces at the end of the line is used to make a linebreak to have the link on a new line.

File: macroquad-introduction-book-english/src/README.md [hunk 2]
```diff
@@ -5,11 +5,11 @@
 This guide is written by Olle Wreede at [Agical](https://www.agical.se/).
 
 <div class="onlyprint">
-This guide is available online at the following address:  
+This guide is available online at the following address:
 https://mq.agical.se/
 </div>
 
-The source code for all chapters of this book is available here:  
+The source code for all chapters of this book is available here:
```
Comment (ollej) [inline macroquad-introduction-book-english/src/README.md:12]: Same newline issue

File: macroquad-introduction-book-english/src/ch6-shooting.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ the speed is set to twice that of the circle.
 {{#include ../../my-game/examples/shooting.rs:shoot}}
 ```
 
-```admonish notes
+```admonish note title="Please note!"s
```
Comment (ollej) [inline macroquad-introduction-book-english/src/ch6-shooting.md:43]: Accidental "s" at the end. Causes an issue when rendering this note.

**Review by ollej — COMMENTED** [review]
Looks good to me! I prefer the shorter ones, but they might be too hard to write.

File: macroquad-introduction-book-english/src/README.md (hunk 1 above)
Comment (PEZ) [reply macroquad-introduction-book-english/src/README.md]: I now added VS Code config to stop me from trimming spaces. =)

File: macroquad-introduction-book-english/src/README.md (hunk 2 above)
Comment (PEZ) [reply macroquad-introduction-book-english/src/README.md]: Now fixed

File: macroquad-introduction-book-english/src/ch6-shooting.md (hunk 3 above)
Comment (PEZ) [reply macroquad-introduction-book-english/src/ch6-shooting.md]: Now fixed. Interesting that admonish is fine with `notes`. But, yeah, I should have used a bit more surgical regexp!

**Pull request merged**